data_model/clusters/BallastConfiguration.xml

```diff
@@ -124,12 +124,12 @@ Davis, CA 95616, USA
       <access read="true" readPrivilege="view"/>
       <mandatoryConform/>
     </attribute>
-    <attribute id="0x0030" name="LampType" type="string" default="empty string">
+    <attribute id="0x0030" name="LampType" type="string" default="&quot;">
```

Comment:
This looks like a bug in the scraper, no?  The spec literally says "empty string"; having a single double-quote as the default value looks very wrong.

Reply:
yes, it is a scraper bug (or at least spec text bug).

Reply:
created https://github.com/csa-data-model/projects/issues/387